src/App.tsx

```diff
@@ -1,26 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [state, setState] = useState({
+    perPage: 5,
+    currentPage: 1,
+  });
```

Comment:
I guess it would be better and easier to have just two different states. There is no need to store them together
```suggestion
  const [perPage, setPerPage] = useState(5);
  const [currentPage, setCurrentPage] = useState(1);
```